apps/10address-book-ajax/script.js

```diff
@@ -1,5 +1,52 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    // You code here
+    $.ajax('https://reqres-api.herokuapp.com/api/users', {
+        success: function(response) {
+            response.forEach(function(users) {
+                var str = "<tr>" +
+                    "<td>" + users.id + "</td>" +
+                    "<td>" + users.first_name + "</td>" +
+                    "<td>" + users.last_name + "</td>" +
+                    "<td>" + "<a href='#' data-id= '" + users.id + "'> view</a>" + "</td>" +
+                    "</tr>";
+
+                $("tbody").append(str);
+            });
+            individiual();
+
+        }
+
+    });
+
+
+    function individiual() {
+        $("td").on('click', 'a', function(event) {
+            event.preventDefault();
+
+            var url = "https://reqres-api.herokuapp.com/api/users/"
+
+            var dataID = $(event.target).data("id");
```

Comment:
Your tests passed but I would recommend using `$(this)`  instead of `event.target` here.